src/MAUI/Maui.Samples/Samples/UtilityNetwork/CreateLoadReport/CreateLoadReport.xaml

```diff
@@ -0,0 +1,44 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage x:Class="ArcGIS.Samples.CreateLoadReport.CreateLoadReport"
+             xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Maui;assembly=Esri.ArcGISRuntime.Maui"
+             xmlns:toolkit="using:Esri.ArcGISRuntime.Toolkit.Maui">
+    <Border Style="{DynamicResource EsriSampleControlPanel}">
+        <VerticalStackLayout Spacing="5">
+            <Label Text="Select a phase using the dropdown, add, then run report." />
```

Comment:
Nitpick `Add phases from the dropdown list then run the report.` 

More concise if you think it's work changing it?